platform/platform_tests_support/scoped_file.cpp

```diff
@@ -22,16 +24,35 @@ ScopedFile::ScopedFile(string const & relativePath)
     : m_fullPath(my::JoinFoldersToPath(GetPlatform().WritableDir(), relativePath)),
       m_reset(false)
 {
+  try
```

Comment:
This block of code looks suspicious - are you sure that we need to do that?

Reply:
Please, call the second ctor with empty contents.

Reply:
I've changed the constructors.